draft-ietf-rtcweb-jsep.xml

```diff
@@ -2118,9 +2118,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>An "a=fingerprint" line for each of the endpoint's
               certificates, as specified in
-              <xref target="RFC8122"></xref>, Section 5; the digest
-              algorithm used for the fingerprint MUST match that used
-              in the certificate signature.</t>
+              <xref target="RFC8122"></xref>, Section 5.</t>
```

Comment:
Since we may need to support multiple hash algos for hash agility, we should make it clear that we could end up with more a=fingerprint lines than certs, i.e. we might need some sort of "for each desired digest algorithm".